openstack/ironic/values.yaml

```diff
@@ -33,6 +33,12 @@ debug: "True"
 
 kos_conductor: true
 
+vault:
+  region: qa-de-1
+
+netbox:
+  url: https://netbox.global.cloud.sap
```

Comment:
Can you remove that please? I'd rather avoid having our internal urls in public repositories.
It isn't really a major issue, just stylistic.